stories/spectra/tga.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from '@storybook/react';
+import React from 'react';
+
+import { Axis, LineSeries, Plot } from '../../src';
+import data1 from '../data/tga1.json';
+import data2 from '../data/tga2.json';
+
+export default {
+  title: 'Experimental spectra/TGA',
+} as Meta;
+
+export function TgaExample() {
+  return (
+    <Plot
+      width={1400}
+      height={540}
+      margin={{ bottom: 45, left: 40, top: 40, right: 40 }}
+      viewportStyle={{ stroke: 'black' }}
+    >
+      <LineSeries
+        data={data1}
+        xAxis="x"
+        yAxis="y"
+        lineStyle={{ stroke: 'red' }}
```

Comment:
you could remove both strokes, by default is giving these colors https://github.com/d3/d3-scale-chromatic#schemeSet1